src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider whether `AbstractFigure` should be an abstract class or an interface based on its purpose and usage in the context of this task.